src/pages/Donations/MobileTable.tsx

```diff
@@ -65,17 +65,6 @@ export default function MobileTable({
                 <Row title="Amount">{humanize(row.amount, 3)}</Row>
                 <Row title="USD Value">{`$${humanize(row.usdValue, 2)}`}</Row>
                 <Row title="TX Hash">{row.hash}</Row>
-                <Row title="Status">
-                  <div
-                    className={`${
-                      row.donationFinalized
```

Comment:
Redundant column, it's apparent which donations are displayed by selecting the appropriate tab.